.github/workflows/main.yaml

```diff
@@ -125,11 +127,3 @@ jobs:
           git config --global user.email "actions@github.com"
           git config --global user.name "GitHub Actions"
           bazel run "//:mkdocs" -- gh-deploy --force
-
-      ########################################
-      # Clean up runner
-      ########################################
-      - uses: ./.github/workflows/manage-runner-post
```

Comment:
We could skip this step if running on our custom runners, instead of deleting the workflow completely. It was helping a lot on upstream GH runners and we might want to be able to run jobs there as well.

Reply:
If we step in the way of self hosted runners i doubt it that we will want to revert back. Maintaining legacy pipeline will be hard and that pipeline will just grow and grow with unreadable if/elses. I think that if we want to make a turn towards self-hosted we shouldn't look into maintaining both